src/main/play-doc/operation/Install.md

```diff
@@ -202,12 +221,17 @@ ConductR supports running applications and services within Docker. If you plan o
 
 _Repeat each step in this section also on the `172.17.0.2` and `172.17.0.3` machine._
 
+For a three node cluster, we recommend that you install both the Core and Agent on all three nodes.
+For larger clusters, 3 core-only nodes can manage many agent-only nodes. An Agent must be install for a node to execute bundles.
+
 First ensure that the following ports are available between the machines forming the cluster:
 
 * 9004 - Akka remoting for ConductR
 * 9006 - Bundle streaming between ConductR nodes
 * 10000 to 10999 - the default range of ports allocated to bundle component endpoints
 
+On most systems `netstat` can be used to check port status, for example: `netstat -an | grep 9004` to check port `9004`.
```

Comment:
Won 't this only work if conductr is already installed? So a useful test for after install, but not before. Or am I missing something?